koku/masu/processor/tasks.py

```diff
@@ -583,77 +642,22 @@ def update_summary_tables(  # noqa: C901
         set_summary_timestamp(ManifestState.FAILED, manifest_id)
         raise ex
 
-    if provider_type != Provider.PROVIDER_OCP:
```

Comment:
586 to 597 here is redundant code now that we always run cost model updates regardless of a cost model existing now. Also tweaking line 655 below should make this simpler.